paddle/fluid/pybind/eager_method.cc

```diff
@@ -1827,6 +1827,32 @@ static PyObject* tensor_method_get_map_tensor(TensorObject* self,
   EAGER_CATCH_AND_THROW_RETURN_NULL
 }
 
+PyDoc_STRVAR(tensor_method_nnz__doc__,
+             R"DOC(nnz($self, /)
+--
+
+**Notes**:
+    **This API is only available for SparseCooTensor/SparseCsrTensor.**
```

Comment:
```suggestion
Note:
    **This API is only available for SparseCooTensor/SparseCsrTensor.**
```

NOTE都改成这种形式吧，官网渲染会有效果